frontend/public/components/dashboard/project-dashboard/project-dashboard.tsx

```diff
@@ -9,25 +10,53 @@ import { UtilizationCard } from './utilization-card';
 import { InventoryCard } from './inventory-card';
 import { ActivityCard } from './activity-card';
 import { ProjectDashboardContext } from './project-dashboard-context';
+import { Firehose, FirehoseResult } from '../../utils';
+import { getQuotaResourceTypes, hasComputeResources } from '../../resource-quota';
+import { ResourceQuotaCard } from './resource-quota-card';
 
 const mainCards = [{ Card: StatusCard }, { Card: UtilizationCard }];
 const leftCards = [{ Card: DetailsCard }, { Card: InventoryCard }];
 const rightCards = [{ Card: ActivityCard }];
 
-export const ProjectDashboard: React.FC<ProjectDashboardProps> = ({ obj }) => {
+const ProjectDashboard_: React.FC<ProjectDashboardProps> = ({ obj, resources }) => {
   const context = {
     obj,
   };
 
+  const resourceQuotas = _.get(resources, 'resourceQuotas.data', []);
+  const quotasCards = resourceQuotas
+    .filter((rq) => hasComputeResources(getQuotaResourceTypes(rq)))
+    .map((rq) => ({
+      Card: () => <ResourceQuotaCard resourceQuota={rq} />,
+    }));
```

Comment:
Does it work if you use...?

```ts
    .map((rq: K8sResourceKind) => <ResourceQuotaCard key={rq.metadata.uid} resourceQuota={rq} />);
```

It's better to avoid the anonymous component since it's a new component every render, which afaik will unmount/remount the component every time. We've had performance issues with this in the past.